virus/sedimentum_occultus.md

```diff
@@ -0,0 +1,111 @@
+
+### Virus Genome
+
+Still Image:
+
+![](img/sedimentum_occultus/figure_1.png)
+
+[Interactive Version](https://www.ncbi.nlm.nih.gov/projects/sviewer/?id=lcl|1&tracks=[key:sequence_track,name:Sequence,display_name:Sequence,id:STD649220238,category:Sequence,annots:Sequence,ShowLabel:false,ColorGaps:false,shown:true,order:0][key:gene_model_track,name:U2EZpwA2z898C,display_name:(U)%20ORFfinder_12.4.72257490,id:U2EZpwA2z898C,data_key:dcrjGxXIwunu5vTuxffK4Jmdm5yTnr-Ut4y5piyrKgnqPwqyJwYWXVoPZHYxamhyelotW25FflRkTnRRQnx7fltOfQ,dbname:NetCache,category:ORFfinder,annots:ORFfinder_12.4.72257490_UUD1701692577DUU_gene_model,Options:MergeAll,CDSProductFeats:false,NtRuler:true,AaRuler:true,HighlightMode:2,ShowLabel:true,highlights_color:#0000ff,shown:true,order:1]&key=_kFokJ5DSWJlbX9lTnxBaxIWEBcYFTQfPAcyLacgoYJitBa9sNaB6w-kRN0RwUjZWvEN8E7uXv9E5VT6Ytdb1XvlXQ&mk=3479:3502|C|red,3797:3832|A|0000FF,3584:3625|B|00FF00,2969:8341|ORF1|black|9,247:2679|ORF2|333333|1,3050:4282|RdRp|FF00FF&noguess_assm=1&spacing=1&v=1:8971&c=null&gflip=1&select=null&slim=0&toolbar=-d) (may not work)
+
+**Figure 1:** Genome of *Sedimentum occultus*. The ORF1 region (in black) shows the location of ORF1, which contains the nucleotide sequence for RdRp. The RdRp region (in pink) shows the location of the RdRp nucleotide sequence that is directly translated into RdRp. Within this region are the motifs A, B, and C (in blue, green, and red, respectively). The ORF2 region (in black) shows the location of ORF2, which contains nucleotide sequences for capsid proteins.
+
+### RdRp Structure
+
+![](img/sedimentum_occultus/figure_2.png)
```

Comment:
For each Figure, please add an inline Title (add `Figure 2` between square brackets)